lib/cryptoexchange/exchanges/gate/services/order_book.rb

```diff
@@ -0,0 +1,44 @@
+module Cryptoexchange::Exchanges
+  module Gate
+    module Services
+      class OrderBook < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Gate::Market::API_URL}/orderBook/#{market_pair.base}_#{market_pair.target}"
```

Comment:
base/target needs to be in reverse after fixing

Reply:
target/base correction